feature-configs/cn-ran-overlays/ran-profile/cluster/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  # Note - should be deleted in CU-only cluster
+  # PTP operator deployment
+  - ../../../deploy/ptp
+  # PTP profiles
+  - du-common/ptp
+
+  # SR-IOV deployment (Networks are deployed as a part of site configuration)
+  - ../../../deploy/sriov
+  # SR-IOV network node policies
+  - ../example/sriov
```

Comment:
Would it make sense to decoulpe example from cluster?
I feel like example should be a layer on top / on the side of cluster, as opposed to a base of it.
In this way, the deployment will be the same but we'll be able to add another layer for a different set of hw configurations.